db/migrate/20221117134008_drop_document_sources.rb

```diff
@@ -0,0 +1,5 @@
+class DropDocumentSources < ActiveRecord::Migration[7.0]
+  def change
+    drop_table :document_sources
```

Comment:
This should stop the Rubocop complaints by providing a way for this migration to rollback.

```suggestion
    drop_table :document_sources do |t|
      t.integer "document_id"
      t.string "url", null: false
      t.integer "import_id"
      t.integer "row_number"
      t.string "locale", default: "en"
      t.index ["document_id"], name: "index_document_sources_on_document_id"
      t.index ["url"], name: "index_document_sources_on_url", unique: true
    end
```